arviz/plots/densityplot.py

```diff
@@ -86,36 +86,36 @@ def plot_density(
     hdi_markers : str
         A valid `matplotlib.markers` like 'v', used to indicate the limits of the highest density
         interval. Defaults to empty string (no marker).
-    shade : Optional[float]
+    shade : float, optional
         Alpha blending value for the shaded area under the curve, between 0 (no shade) and 1
         (opaque). Defaults to 0.
-    bw: Optional[float or str]
+    bw : float or str, optional
         If numeric, indicates the bandwidth and must be positive.
         If str, indicates the method to estimate the bandwidth and must be
         one of "scott", "silverman", "isj" or "experimental" when `circular` is False
         and "taylor" (for now) when `circular` is True.
         Defaults to "default" which means "experimental" when variable is not circular
         and "taylor" when it is.
-    circular: Optional[bool]
+    circular : bool, optional
         If True, it interprets the values passed are from a circular variable measured in radians
         and a circular KDE is used. Only valid for 1D KDE. Defaults to False.
     grid : tuple
         Number of rows and columns. Defaults to None, the rows and columns are
         automatically inferred.
-    figsize : Optional[Tuple[int, int]]
+    figsize : int, optional
         Figure size. If None it will be defined automatically.
-    textsize: Optional[float]
+    textsize : float, optional
         Text size scaling factor for labels, titles and lines. If None it will be autoscaled based
         on ``figsize``.
-    labeller : labeller instance, optional
+    labeller : Labeller instance, optional
         Class providing the method ``make_label_vert`` to generate the labels in the plot titles.
         Read the :ref:`label_guide` for more details and usage examples.
-    ax: numpy array-like of matplotlib axes or bokeh figures, optional
+    ax : 2D array-like of matplotlib _axes or bokeh _figure, optional
         A 2D array of locations into which to plot the densities. If not supplied, Arviz will create
         its own array of plot areas (and return it).
-    backend: str, optional
+    backend : str, optional
         Select plotting backend {"matplotlib","bokeh"}. Default "matplotlib".
-    backend_kwargs: bool, optional
+    backend_kwargs : bool, optional
```

Comment:
```suggestion
    backend_kwargs : dict, optional
```

That one was also incorrect in the original docstring. Arguments that have `kwargs` in the name neary always take `dict` as input type. If you see some that say something else let me know, they are probably also wrong.